cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js

```diff
@@ -45,20 +44,16 @@ export default {
       Select({ name:'profile.existingRecordType' }).choose(specialMappingProfile.typeValue)
     ]);
     if (specialMappingProfile.typeValue === 'Holdings') {
```

Comment:
Please move values like 'Holdings' and 'Item' in constant or object with this values. It can help to avoid problems with UI name changes in future.

Reply:
Thanks! I did it